### PR TITLE
Fix Hasura Count Aggregation Filter

### LINKED ISSUE #1377: [BUG] Hasura Count Aggregation Filter
**Describe the bug**
In Hasura and may be other similar packages too aggregation doesn't have filters

**To Reproduce**
1. Use Hasura dataProvider
2. Apply filters in list
3. Scroll down to page numbers
4. There will be empty pages without data

**Expected behavior**
Pagination count creates problem resulting in empty pages

**Screenshots**
screenshot-localhost_3000-2021 12 19-13_33_14

**Desktop (please complete the following information):**
 - OS: [macOS]
 - Browser [Chrome]
 - Version [96]
 - Used dataProvider [Hasura]

**Additional context**
We can use variables of filters in main operation in aggregate operation too

### PR DESCRIPTION
**Pull request by narayandreamer**
Aggregation Filters patch
Aggregation Operation Spelling

**Closing issues**

Put `closes #1377` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by omeraplak** [comment]
Hey @narayandreamer ,
Great! Can we merge this PR? Or is it still in WIP status?

**Comment by narayandreamer** [comment]
It's WIP, Could you tell me how to build dist folder in hasura package so that I can test and then remove WIP?

**Comment by omeraplak** [comment]
> It's WIP, Could you tell me how to build dist folder in hasura package so that I can test and then remove WIP?

Ohh okay. Our document about local development,
https://refine.dev/docs/contributing/#running-in-development-mode

If you want to run specific projects you can use Lerna's filters,
https://github.com/lerna/lerna/tree/main/core/filter-options

Don't forget to make sure the path of the project you want to run is in `lerna.json`,
https://github.com/pankod/refine/blob/master/lerna.json#L2

For Hasura example,
```
{
  "packages": [
    "packages/*",
    "examples/dataProvider/hasura"
  ],
  "registry": "https://registry.npmjs.org/",
  "npmClient": "npm",
  "publishConfig": {
    "access": "public"
  },
  "version": "2.4.11"
}
```

example;
```
npm install
npm run bootstrap
npm run build
npm run start -- --scope @pankod/refine --scope @pankod/refine-hasura --scope refine-hasura-example
```

File: packages/hasura/src/index.ts [hunk 1]
```diff
@@ -160,16 +160,24 @@ const dataProvider = (client: GraphQLClient): DataProvider => {
...
+                    operation: aggregateOperation,
                     fields: [{ aggregate: ["count"] }],
+                    variables: {
+                        ...({
+                            where: {
+                                value: hasuraFilters,
+                                type: hasuraFiltersType,
+                            },
+                        }),
+                    },
```
Comment (omeraplak) [inline packages/hasura/src/index.ts:172]:
```suggestion
                    variables: {
                        where: {
                            value: hasuraFilters,
                            type: hasuraFiltersType,
                        },
                    },
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

**Comment by omeraplak** [comment]
Hey @narayandreamer ,
We've tested your PR and works very well. Congrats! Thank you

**Comment by narayandreamer** [comment]
Hi @omeraplak . Sorry I was AFK last week so couldn't update you. Cool it works

**Pull request merged**